scripts/_common.sh

```diff
@@ -14,10 +14,10 @@ pkg_dependencies="\
         "
 
 RUBY_VERSION="2.7.3"
-NODEJS_VERSION="15.11.0"
+NODEJS_VERSION="16"
 
 SOURCE="https://github.com/standardnotes/web"
-COMMIT="e856ce7bfd3a7b357544e43ca343b1ec861debb3"
+COMMIT="90fcac61e0ed58e875648951fd4c257cd77a4165"
```

Comment:
The current used commit is from the latest release: ( 6 month old )
https://github.com/standardnotes/web/releases/tag/3.8.25

Instead of using the latest upstream commit I would say we could use the commit of the lattest tag: ( 16 days old )
https://github.com/standardnotes/web/tree/10.17.0

Than we have at least a fixed version 10.17.0~ynh1

And than I would also change the auto update script from release to tag.
Or what do you think about this?